eos-tech-support/eos-enable-debian-apt

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+echo "This script adds the Debian repository to your apt config."
+echo "The resulting configuration prefers the Endless repository where"
+echo "possible, even if a package in Endless is older than in Debian."
+echo "However, but packages not found in the Endless repo will be automatically"
+echo "downloaded from Debian".
+echo
+echo "Although this may work fine in some cases, you're likely to find issues"
+echo "where a package in Debian is somehow incompatible with Endless packages"
+echo "and this will most frequently appear as dependency resolution issues."
+echo "As such, this is an unsupported, developer-only tool."
+echo
+echo "Additionally, apt will only work after you have taken the usual steps of"
+echo "making it available (e.g. eos-convert-system, eos-dev-unlock, etc)"
+echo
+
+if [[ $# < 1 ]]; then
+	echo "Usage: $0 <DEB_DISTRO>"
+	echo "DEB_DISTRO example values: stable, testing, unstable"
+	exit 1
+fi
+
+distro=$1
+
+cat > /etc/apt/preferences.d/eos-enable-debian-apt <<EOF
+Package: *
+Pin: release o=Endless
+Pin-Priority: 600
+
+Package: *
+Pin: release o=Debian
+Pin-Priority: 400
```

Comment:
On a related note, it would be nice to add a comment above this block explaining why the 400 was picked (make it below the default 500 so debian packages are only installed if they aren't on Endless or the debian release was specified).